doc.html

```diff
@@ -4,230 +4,90 @@
-  <tr><td>STAT3</td><td>SL3315_SL3319</td></tr>
-  <tr><td>Hif1a</td><td>SL10572_SL10566</td></tr>
-  <tr><td>Etv6</td><td>SL13368_SL13364</td></tr>
-  <tr><td>Jmjd3</td><td>SL12384_SL12382</td></tr>
-  <tr><td>BATF-Th0</td><td>SL3039_SL3038</td></tr>
-  <tr><td>BATF-Th17</td><td>SL3033_SL3036</td></tr>
-  <tr><td>cMaf-Th0</td><td>SL4424_SL4425</td></tr>
-  <tr><td>cMaf-Th17</td><td>SL3031_SL2871</td></tr>
-  <tr><td>Fosl2-Th0</td><td>SL6500_SL6499</td></tr>
-  <tr><td>Fosl2-Th17</td><td>SL6498_SL6497</td></tr>
-  <tr><td>IRF4-Th0</td><td>SL1044_SL1043</td></tr>
-  <tr><td>IRF4-Th17</td><td>SL971_SL970</td></tr>
-  <tr><td>p300-Th0</td><td>SL1948_SL1947</td></tr>
-  <tr><td>p300-Th17</td><td>SL1041_SL972</td></tr>
-  <tr><td>RORg-Th0</td><td>SL3779_SL3778</td></tr>
-  <tr><td>RORg-Th17</td><td>SL1944_SL1943</td></tr>
-  <tr><td>STAT3-Th0</td><td>SL3780_SL3778</td></tr>
-  <tr><td>STAT3-Th17</td><td>SL973_SL972</td></tr>
-  <tr><td>RNA-Seq-1h</td><td>SL1851</td></tr>
-  <tr><td>RNA-Seq-3h</td><td>SL1852</td></tr>
-  <tr><td>RNA-Seq-6h</td><td>SL1853</td></tr>
-  <tr><td>RNA-Seq-9h</td><td>SL1854</td></tr>
-  <tr><td>RNA-Seq-16h</td><td>SL8355</td></tr>
-  <tr><td>RNA-Seq-24h</td><td>SL1857</td></tr>
-  <tr><td>RNA-Seq-48h</td><td>SL1858</td></tr>
-  <tr><td>FAIRE-Seq-IRF4+</td><td>SL14406</td></tr>
-  <tr><td>FAIRE-Seq-IRF4-</td><td>SL14408</td></tr>
-  <tr><td>FAIRE-Seq-Batf+</td><td>SL14523</td></tr>
-  <tr><td>FAIRE-Seq-Batf-</td><td>SL14526</td></tr>
-</table>
-</li>
-</ul>
+  <script type="text/javascript" src="bower_components/jquery/dist/jquery.min.js"></script>
+  <script type="text/javascript" src="bower_components/jquery-ui/jquery-ui.min.js"></script>
+  <script type="text/javascript" src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
 
+  <link rel="stylesheet" href="bower_components/jquery-ui/themes/smoothness/jquery-ui.min.css"/>
+  <link rel="stylesheet" href="bower_components/jquery-ui/themes/smoothness/theme.css"/>
+  <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css"/>
+  <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap-theme.min.css"/>
+</head>
+<body>
+  <div class="container">
+    <h2> Genotet - Help Document </h2>
+    <p>For more information, contact <a href="mailto:bowen.yu@nyu.edu">Bowen</a>, <a href="mailto:jiamingdong@nyu.edu">Jiaming</a> or <a href="mailto:ling.yang@nyu.edu">Liana</a>. </p>
+    <p>Note: This document is currently only for data format of data uploading.</p>
+    <h3>Network</h3>
+    <ul>
+      <li>Currently, we accept *.tsv file for network uploading.</li>
+      <li>The first line contains N + 2 space seperated strings, containing 2 table head of "regulator", "target" and N names of values.</li>
+      <li>Following M lines, containing M edges from regulators to targets, each line contains N + 2 elements. They are 1 regulator, 1 target and N float values.</li>
+      <li>It is like
+      <pre>
```

Comment:
Could you please correct the indentation here? pre need to be indented 2 more spaces than its parent li. The closing </pre> need to be on a new line.